deflect/NetworkListener.h

```diff
@@ -55,14 +56,17 @@ class NetworkListener : public QTcpServer
 
 public:
     /** The default port number used for Stream connections. */
-    DEFLECT_API static const int defaultPortNumber_;
+    DEFLECT_API static const int defaultPortNumber;
```

Comment:
And now I realise that defaultPortNumber is also redefined in the Socket class, but it should only be defined in one place... I'm not sure if it should stay here, or be moved in types.h? Same question for the serviceName...?
Also, I think this class should be renamed from deflect::NetworkListener to deflect::Server, but that would take us too far for this code review...